frmts/georaster/cpl_vsil_ocilob.cpp

```diff
@@ -93,62 +92,101 @@ class VSIOCILobHandle : public VSIVirtualHandle
 // Implementation                                             WSIOCILobFSHandle
 // ****************************************************************************
 
-// ----------------------------------------------------------------------------
-//                                                          WSIOCILobFSHandle()
-// ----------------------------------------------------------------------------
-
-WSIOCILobFSHandle::WSIOCILobFSHandle()
-{
-    poStatement  = nullptr;
-    phLocator    = nullptr;
-    poConnection = nullptr;
-}
-
 // -----------------------------------------------------------------------------
-//                                                          ~WSIOCILobFSHandle()
+//                                                          ParseIdentificator()
 // -----------------------------------------------------------------------------
 
-WSIOCILobFSHandle::~WSIOCILobFSHandle()
+char** WSIOCILobFSHandle::ParseIdentificator( const char* pszFilename )
```

Comment:
you could return a CPLStringList instead of a char**. That would save you all the explicit CSLDestroy() calls

Reply:
It is good to know this. I would like to keep the change minimal for now.